fv3core/stencils/ytp_v.py

```diff
@@ -121,37 +120,47 @@ def _compute_stencil(
         flux = _get_flux(v, courant, rdy, bl, br)
 
 
-def compute(c: FloatField, v: FloatField, flux: FloatField):
-    """
-    Compute flux of kinetic energy in y-dir.
+class YTP_V:
```

Comment:
I'm good with capitalizing the existing names in this first iteration to keep the naming straightforward. When we make these class names readable, we should use CamelCase convention without underscores to keep in line with pep 8.